src/applications/mhv/medical-records/containers/Allergies.jsx

```diff
@@ -136,10 +136,15 @@ const Allergies = props => {
     }
     if (allergies?.length === 0) {
       return (
-        <div className="vads-u-margin-bottom--3">
-          <va-alert background-only status="info">
-            You don’t have any records in Allergies
-          </va-alert>
+        <div
+          className="record-list-item vads-u-border-color--gray-light vads-u-border--0 vads-u-background-color--gray-lightest card"
+          data-testid="record-list-item"
+        >
+          <h2 className="vads-u-font-size--base vads-u-font-weight--normal">
+            <p className="page-description">
+              There are no allergies or reactions in your VA medical records.
+            </p>
+          </h2>
```

Comment:
Something here is still causing the font to be larger than other text on the page, despite the classes you added.